File: PokerProtocol/src/protocol/commands/game/TableClosedCommand.java
package protocol.commands.game;

import java.util.StringTokenizer;

import protocol.commands.ICommand;

public class TableClosedCommand implements ICommand
{
    public static String COMMAND_NAME = "gameTABLE_CLOSED";
    
    public TableClosedCommand(StringTokenizer argsToken)
    {
    }
    
    public TableClosedCommand()
    {
    }
    
    @Override
    public String encodeCommand()
    {
        final StringBuilder sb = new StringBuilder();
        sb.append(TableClosedCommand.COMMAND_NAME);
        return sb.toString();
    }
}

File: trunk/C#/Com.Ericmas001.Game.Poker.Protocol.Commands/Game/TableClosedCommand.cs
using System;
using System.Collections.Generic;
using System.Text;
using Com.Ericmas001;
using Com.Ericmas001.Net.Protocol.JSON;

namespace Com.Ericmas001.Game.Poker.Protocol.Commands.Game
{
    public class TableClosedCommand : AbstractJsonCommand
    {
        public static string COMMAND_NAME = "gameTABLE_CLOSED";
    }
}
